torch/csrc/jit/codegen/cuda/iter_visitor.cpp

```diff
@@ -52,15 +82,21 @@ void remove_visited(
 void IterVisitor::traverseFrom(
     Fusion* const fusion,
     const std::vector<Val*>& from,
-    bool traverseAllPaths) {
+    bool traverseAllPaths,
+    bool respectComputeAt) {
+  // TODO: Remove the temporary getenv
```

Comment:
are you planning to remove the getenv before merging, or is this a TODO for sometime in the future?

Reply:
Yes, it's a TODO that I will do when making this PR non-draft.